lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'menu'
+
+class Order
+  def initialize(*choice, quantities, user_total)
+    @choice = choice
+    @quantities = quantities
+    @user_total = user_total
+    @menu = Menu.new
+  end
+
+  def calculate_price
+    convert_dishes_to_prices
+    @price = (@prices.zip(@quantities)).map { |x, y| x * y }
+    @price.sum
+  end
+
+  def convert_dishes_to_prices
+    @prices = []
+    @choice.each { |food| @prices << @menu.menu[food] }
+  end
+
+  def verified?
+    calculate_price == @user_total
+    send_text
+  end
+
+  private
```

Comment:
I like that send_text is private. I think some of the other methods could be too